[2021-11-26 13:19:51,449] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 manual__2021-11-26T20:19:40.360596+00:00 [queued]>
[2021-11-26 13:19:51,451] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 manual__2021-11-26T20:19:40.360596+00:00 [queued]>
[2021-11-26 13:19:51,451] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-26 13:19:51,451] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-26 13:19:51,451] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-26 13:19:51,456] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): runme_0> on 2021-11-26 20:19:40.360596+00:00
[2021-11-26 13:19:51,459] {standard_task_runner.py:52} INFO - Started process 15579 to run task
[2021-11-26 13:19:51,470] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_0', 'manual__2021-11-26T20:19:40.360596+00:00', '--job-id', '8', '--raw', '--subdir', '/Users/danielcollins/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/var/folders/9_/3lc9lq1570l2_y2nsyxbktvw0000gn/T/tmpalnk7bdt', '--error-file', '/var/folders/9_/3lc9lq1570l2_y2nsyxbktvw0000gn/T/tmp2rkbk15_']
[2021-11-26 13:19:51,471] {standard_task_runner.py:77} INFO - Job 8: Subtask runme_0
[2021-11-26 13:19:51,506] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.runme_0 manual__2021-11-26T20:19:40.360596+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-11-26 13:19:51,525] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_0
AIRFLOW_CTX_EXECUTION_DATE=2021-11-26T20:19:40.360596+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-26T20:19:40.360596+00:00
[2021-11-26 13:19:51,526] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/9_/3lc9lq1570l2_y2nsyxbktvw0000gn/T
[2021-11-26 13:19:51,527] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "example_bash_operator__runme_0__20211126" && sleep 1']
[2021-11-26 13:19:51,534] {subprocess.py:85} INFO - Output:
[2021-11-26 13:19:51,541] {subprocess.py:89} INFO - example_bash_operator__runme_0__20211126
[2021-11-26 13:19:52,555] {subprocess.py:93} INFO - Command exited with return code 0
[2021-11-26 13:19:52,586] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_0, execution_date=20211126T201940, start_date=20211126T201951, end_date=20211126T201952
[2021-11-26 13:19:52,618] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-26 13:19:52,644] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
